WordPressUtils/src/main/java/org/wordpress/android/util/HtmlUtils.java

```diff
@@ -114,7 +125,11 @@ public static String stripScript(final String text) {
     }
 
     /**
-     * an alternative to Html.fromHtml() supporting <ul>, <ol>, <blockquote> tags and replacing EmoticonsUtils with Emojis
+     * An alternative to Html.fromHtml() supporting "ul" (&lt;ul&gt;), "ol" (&lt;ol&gt;), "blockquote" (&lt;blockquote&gt;)
```

Comment:
We should use the `(@code)` notation here and in other places where html tags are used in comments, it makes things readable in the comment and in the generated html. For instance here: `{@code <ul>}, {@code <ol>},  {@code <ol>},  {@code <blockquote>},`